java/ddlogapi.c

```diff
@@ -13,6 +13,74 @@
 #include "ddlog.h"
 #include "ddlog_log.h"
 
+/* Error message returned by DDlog. */
+_Thread_local char* err_msg = NULL;
```

Comment:
Is this standard? I could not find documentation for it on the web.

Reply:
It's C11: https://en.cppreference.com/w/c/language/storage_duration

I checked that it works with gcc and clang.